mods/doors/init.lua

```diff
@@ -634,10 +642,14 @@ function doors.register_trapdoor(name, def)
 	local def_opened = table.copy(def)
 	local def_closed = table.copy(def)
 
-	def_closed.node_box = {
-		type = "fixed",
-		fixed = {-0.5, -0.5, -0.5, 0.5, -6/16, 0.5}
-	}
+	if def.closed and def.opened then
```

Comment:
```suggestion
	if def.closed then
```

Reply:
But we need to be sure that they have both models specified. Or do we? I wonder if we should allow the mod to use either one of them.

Reply:
It would be a bit awkward if a modder wants to specify one first to test and then he wonders why it doesn't work.

Reply:
This is fine as is, both nodeboxes should be present to not use the default.

Reply:
But then this should be documented to avoid confusion.